tests/snippets/builtins_module.py

```diff
@@ -0,0 +1,28 @@
+from testutils import assertRaises
+
+assert '__builtins__' in globals()
+# assert type(__builtins__).__name__ == 'module'
+with assertRaises(AttributeError):
+    __builtins__.__builtins__
+
+__builtins__.x = 'new'
+assert x == 'new'
+
+exec('assert "__builtins__" in globals()', dict())
+exec('assert __builtins__ == 7', {'__builtins__': 7})
+exec('assert not isinstance(__builtins__, dict)')
+exec('assert isinstance(__builtins__, dict)', {})
+
+namespace = {}
+exec('', namespace)
+assert namespace['__builtins__'] == __builtins__.__dict__
+
+# with assertRaises(NameError):
```

Comment:
What is the issue with this test? Can this test be enabled?

Reply:
In RustPython, `exec('print()')` always succeed without builtins injection (original state).
Because this patch didn't touch anything about name resolving, this code still works - so the test is failed.